Tools/CI/regenerate.sh

```diff
@@ -0,0 +1,2 @@
+cd $(dirname "$0")/../../
+dotnet run --project Tools\CI\UnityMobileNotifications.Cookbook.csproj
```

Comment:
Shouldn't UnityMobileNotifications.Cookbook.csproj be added to repo ?

Reply:
Good catch.